tests/testthat/test-RInterface.R

```diff
@@ -6,6 +6,55 @@
 library(BinSeg)
 set.seed(300)
 
+psd <- function(x) {
+  sqrt(sum((x - mean(x))^2) / (length(x)))
+}
+
+fixed_mean_psd <- function(x, mean){
+  sqrt(sum((x - mean)^2) / (length(x)))
+}
+
+check_resid <- function(object, ncpts=nrow(object@models_summary)){
+
+  dist <- dist(object)
+  obj_coef <- coef(object, ncpts)
+  resids <- resid(object)
+  data <- object@data
+
+  if (dist == "mean_norm"){ # Cost function per segment.
+    resid_func <- function(start, end){
+      tmp <- resids[start:end] * sd(data)
+      tmp <- tmp + mean(data[start:end])
```

Comment:
@diego-urgell please use a more informative name than `tmp` if possible.
Also please avoid re-writing /changing the same variable `tmp <- tmp + something`
both are very confusing for others who read your code.